bitrise.yml

```diff
@@ -426,6 +426,18 @@ workflows:
     - swiftlint-extended@1:
         inputs:
         - linting_path: "$BITRISE_SOURCE_DIR"
+    - deploy-to-bitrise-io@1.9: {}
+    - script@1:
+        inputs:
+        - content: |-
+            set -e
+            echo "$SWIFTLINT_REPORT"
+            if [ ! -z "$SWIFTLINT_REPORT" ] ;then
```

Comment:
a really useful tool to have is `shellcheck` to help with some best practices like the use of `! -z`. it's best to use `-n STRING` instead. rule: https://www.shellcheck.net/wiki/SC2236

Reply:
Let me add this also to the ticket I commented below, so we stablish a common best practices doc and follow the rules for the whole bitrise.yml file